From ec991533953db675fe5acbb4932cce5585d7ee62 Mon Sep 17 00:00:00 2001
From: chandavath <rcrajagopal@habana.ai>
Date: Mon, 26 Feb 2024 06:39:22 +0200
Subject: [PATCH] [SW-176338] gaudi3: added pcie functions in exports elf

to perform pcie config in erom,
the following functions from arcboot are exported in exports.elf.

pcie functions are pcie_phy_para_sel_set, get_sris_mode_from_strap,
pcie_phy_set_lane_refclk_mode, pcie_phy_set_pll_short_lock,
get_srns_mode_from_strap, pcie_phy_ssc_en_set,
pcie_phy_set_sram_ext_ld_done, pcie_set_pcie_device_vendor_id

Change-Id: I4acd86b224f32999fe51a3a055533b2ff4c60961
Signed-off-by: chandavath <rcrajagopal@habana.ai>
---

diff --git a/arch/gaudi3/arc/exports.elf b/arch/gaudi3/arc/exports.elf
index 2e6517d..2f463cb 100644
--- a/arch/gaudi3/arc/exports.elf
+++ b/arch/gaudi3/arc/exports.elf
Binary files differ
diff --git a/arch/gaudi3/arc/exports_onix.elf b/arch/gaudi3/arc/exports_onix.elf
index de2eb48..f1249da 100644
--- a/arch/gaudi3/arc/exports_onix.elf
+++ b/arch/gaudi3/arc/exports_onix.elf
Binary files differ
